src/filtrering/filtrering-filter/filterform/checkbox-filterform.tsx

```diff
@@ -54,27 +44,39 @@ function CheckboxFilterform({
             {harValg && (
                 <div className={classNames('checkbox-filterform__valg', className)}>
                     <Grid columns={gridColumns}>
-                        <CheckboxGroup legend="" value={checkBoxValg} size="small">
-                        {Object.entries(valg).map(([filterKey, filterValue]) =>
-                            tooltips && tooltips[filterKey] ? (
-                                <Tooltip
-                                    content={tooltips[filterKey]}
-                                    placement="right"
-                                    offset={-130}
-                                    maxChar={999}
-                                    key={`tooltip-${filterKey}`}
-                                >
-                                    <Checkbox key={filterKey} value={filterKey} onChange={velgCheckBox} data-testid={`filter_${filterKey}`}>
+                        <CheckboxGroup
+                            hideLegend
+                            legend=""
+                            onChange={(filtre: string[]) => endreFiltervalg(form, filtre)}
+                            size="small"
+                            value={checkBoxValg}
+                        >
+                            {Object.entries(valg).map(([filterKey, filterValue]) =>
+                                tooltips && tooltips[filterKey] ? (
+                                    <Tooltip
```

Comment:
It seems that tooltip is not correctly placed with new checkboxes...can you please check that